### PR TITLE
1.17

### PR DESCRIPTION
**Pull request by emilyalexandra**
This isn't clean, there are merge conflicts, the version this targets is a few snapshots outdated, and I'm sure I missed some of the logic because I rushed to just get this usable for my use cases in floralisia, but this should ease willie's porting

### CODE REVIEW

**Review by Hubry — COMMENTED** [review]
Probably should also change the `mc_version` def in the buildscript to the targetted snapshot?

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
```
[the remark this answers is not in the document]
Comment (emilyalexandra) [reply .vscode/launch.json:1]: oh lol I didn't even notice it got caught up, .gitignores always seem to miss .vscode

**Comment by Hubry** [comment]
Looks like CI failed because of Java 8 in the workflow file, I guess this looks good.

**Comment by williewillus** [comment]
Do multiblocks work? (in book and world)

**Comment by emilyalexandra** [comment]
No clue, I didn't test it in earlier snapshots and I can't test easily anymore

**Comment by williewillus** [comment]
applied to 1.17-fabric branch

**Pull request closed without merging**